### PR TITLE
Auto update vs links

### PR DESCRIPTION
**Pull request by orta**
When the site is published to staging, this scripts the NPM API to grab the latest stable version, then looks through the [extensions marketplace](https://marketplace.visualstudio.com/search?term=typescript&target=VS&category=Tools&vsVersion=vs15&subCategory=All&sortBy=Relevance) for an extension by us which has the same version number. If it doesn't it won't change it, and it'll check again on the next deploy.

### CODE REVIEW

**Comment by orta** [comment]
Re: https://github.com/microsoft/TypeScript/issues/32729

File: scripts/updateURLs.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+// @ts-enable
```
Comment (weswigham) [inline scripts/updateURLs.js]: wait. It's `//@ts-check`.

Comment (orta) [reply scripts/updateURLs.js]: Hah, that does explain why it wasn't doing it :D

Comment (orta) [reply scripts/updateURLs.js]: Thanks!

**Comment by uniqueiniquity** [comment]
If the version numbers ever slip out of sync (this happens rarely, but it does happen), will it be a mess to update manually until the following release?

**Comment by orta** [comment]
If the version numbers are messed up it won't change the JSON file, so that can be manually changed 👍

**Review by uniqueiniquity — APPROVED** [review]

**Pull request merged**